keystrokes/cypress/integration/characterCreationModal.spec.js

```diff
@@ -0,0 +1,96 @@
+import { characterDetails } from '../../src/config/constants';
+describe('Tests for CharacterCreationModal', () => {
+	it('authenticates', () => {
+		cy.visit('/');
+		cy.login('ILHEEvO7wmWa0r7xtHqMmmQ3vxe2'); //might want to put this somewhere else
```

Comment:
What is this hash?

Reply:
its the user uid for the test user. its the user in the seed database.